apps/frontend/src/constants/currencies.ts

```diff
@@ -0,0 +1,2 @@
+export const BITCOIN = 'BTC';
+export const USD = 'USD';
```

Comment:
Extract the `XX_RENDER_PRECISION` constants from `apps\frontend\src\app\3_organisms\ZeroLocForm\constants.ts` here as well, they are used in other places than just LoC form